### PR TITLE
RSDK-5386 Update cartographer configs with cloud slam

### PR DESCRIPTION
**Pull request by vijayvuyyuru**
Hey yall, this PR is to update the sample configs in cartographer with the new cloud story and IMU changes.

### CODE REVIEW

File: module/sample_linux.json [hunk 1]
```diff
@@ -1,44 +1,107 @@
...
+                "method": "AngularVelocity",
+                "capture_frequency_hz": 20
+              }
+            ]
+          }
+        }
+      ],
+      "type": "movement_sensor",
+      "attributes": {
+        "serial_path": "/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0"
```
[the remark this answers is not in the document]
Comment (vijayvuyyuru) [reply module/sample_linux.json:29]: I was also not sure whether to put `ttyUSB0` vs the `by_id` I dont have a strong preference either way

[the remark this answers is not in the document]
Comment (kim-mishra) [reply module/sample_linux.json:29]: maybe `"/dev/serial/by-path/<device_ID>" makes sense so anyone looking knows they have to check that

File: module/sample_linux.json [hunk 2]
```diff
@@ -1,44 +1,107 @@
...
+      "model": "viam:lidar:rplidar",
+      "type": "camera",
+      "namespace": "rdk"
     }
   ],
   "modules": [
     {
-      "executable_path": "/usr/local/bin/rplidar-module",
-      "name": "rplidar_module"
+      "version": "0.3.37",
```
Comment (tessavitabile) [inline module/sample_linux.json:59]: I suggest also leaving this for the user to fill in.

File: module/sample_linux.json [hunk 3]
```diff
@@ -1,44 +1,107 @@
...
+      "module_id": "viam:cartographer",
+      "name": "viam_cartographer",
+      "type": "registry"
+    },
+    {
+      "version": "0.1.15",
```
Comment (tessavitabile) [inline module/sample_linux.json:65]: Same here

File: module/sample_linux.json [hunk 4]
```diff
@@ -1,44 +1,107 @@
...
+        }
+      },
+      "name": "slam",
+      "type": "slam",
+      "namespace": "rdk",
+      "model": "viam:slam:cartographer"
     }
   ]
-}
+}
```
Comment (tessavitabile) [inline module/sample_linux.json:107]: Missing newline

**Review by tessavitabile — APPROVED** [review]
LGTM mod comments

**Review by kim-mishra — APPROVED** [review]

**Pull request merged**